src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20211012154859_GeometryData.sql

```diff
@@ -0,0 +1,219 @@
+SET IDENTITY_INSERT geometry ON;
```

Comment:
Out of interest, is this geometry table a predefined table structure specifically for these file types that we get from Geodata sites?  Or did you have to do any magic to get this into the right format?

Reply:
Ahh nice - that's the sort of magic that I didn't know we did until just now!